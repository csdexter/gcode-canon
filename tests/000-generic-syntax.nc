%
/ N10 (you should not see this one)
/ (MSG,neither this one)

O10
(MSG,this is a message)
(this is a comment)

G00 0.000 0.000 0.000

%