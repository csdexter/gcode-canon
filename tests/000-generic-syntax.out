MSG: WAR: Machine servos activated!
MSG: STA: Scanning input for programs (O words)
MSG: this is a message
